pipelines/controller/README.md

```diff
@@ -1,2 +1,80 @@
-Please see
-[this Wiki page](https://github.com/google/fhir-data-pipes/wiki/Try-out-the-FHIR-Pipelines-Controller).
+# FHIR Pipelines Controller
+
+The FHIR Pipelines Controller helps you schedule and manage the transformation
+of data from a [HAPI FHIR server](https://hapifhir.io/) to a collection of
+Apache Parquet files. It uses the FHIR Data Pipes JDBC pipeline to run either
+full or incremental transformations to a Parquet data warehouse.
+
+The FHIR Pipelines Controller only works with HAPI FHIR servers using Postgres.
+You can see
+[an example of configuring a HAPI FHIR server to use Postgres here](https://github.com/hapifhir/hapi-fhir-jpaserver-starter#postgresql-configuration).
+
+## Usage
+
+### Setup
+
+Clone the
+[fhir-data-pipes GitHub repository](https://github.com/google/fhir-data-pipes),
+open a terminal window, and `cd` to the directory where you cloned it. Later
+terminal commands will assume your working directory is the repository root.
```

Comment:
Suggestion: change the assumption and start from the `pipelines/controller` directory instead as I think you have no commands run from the root (in the current version, other than `cd pipelines/controller` below). Then update the paths below to be relative to the `controller` directory, i.e., drop `pipelines/controller`.